README.md

```diff
@@ -28,7 +28,7 @@ To complete the installation of this plugin you will need a ThisData account.
 
 **Now install the plugin**
 
-1. Upload the plugin files to the `/wp-content/plugins/thisdata` directory, or install the plugin through the WordPress plugins screen directly.
+1. Upload the [plugin files](https://github.com/thisdata/thisdata-wordpress/blob/master/dist/thisdata-wordpress.zip) to the `/wp-content/plugins/thisdata` directory, or install the plugin through the WordPress plugins screen directly.
```

Comment:
I think if you add `?raw=true` it'll do the magical auto-download for you. Otherwise it tries to display the zip.